data_structures/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(include)
```

Comment:
I'm thinking the CMakeLists template on the [contributing guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#integrating-cmake-in-a-new-directory) should be used (along with this code), so that it works as expected, I guess? 🤔

Reply:
Can I keep it till I finished the whole `data_structures` reconstruction?

Reply:
It should be added to make sure the CMake builds work properly.

Reply:
Done. I'll push it later.